src/js/services/lang.js

```diff
@@ -11,13 +11,15 @@ const languages = {
     url: 'docs/',
     id: 'globalNavLang-en',
     domEl: getElById('globalNavLang-en'),
+    activeLang: getElById('select-lang-en'),
```

Comment:
Do we need to add this `activeLang` property? We could use the existing`domEl` query and target the image tag within it? In pseudocode:

$(currentLang.domEl > "classname_of_checkmark_image").toggleClass etc.

This way we don't have to add a unique id to every img tag for each language we have.

Reply:
I acknowledge that that the above is a nit.

Reply:
very cool! I wasnt aware that we could do that. Ill make the updates 👍

Reply:
I was having some trouble with the syntax, tried a few other ways including using a template literal but it didnt seem to be able to target the child element. I'm using `$(currentLang.domEl).children('.select-lang-wrap').toggleClass('no-display', false);` and its working nicely 👌